client/components/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@automattic/simple-components",
+  "version": "0.1.0-alpha.1",
+  "description": "React components, as used on WordPress.com",
+  "scripts": {
+    "build": "babel ./index.js --out-dir cjs && babel ./button --out-dir cjs/button && babel ./main --out-dir cjs/main",
```

Comment:
Noting here that I also tried

```
babel --only ./index.js,./button/,./main/ --out-dir cjs
```

but that is unfortunately evaluated as a glob (and apparently broken) since it includes e.g. every path that ends in `button/` or `main`/, and every `index.js` (as opposed to just top-level ones).

Babel 7 [changes](https://github.com/babel/babel/pull/5487) `--only` and `--ignore` args to be evaluated as relative paths instead of globs.